src/consensus/pbft_manager.cpp

```diff
@@ -655,19 +652,15 @@ bool PbftManager::stateOperations_() {
 
   if (is_syncing_()) return true;
 
-  // Get votes
-  votes_ = vote_mgr_->getVerifiedVotes(round, sortition_threshold_, getDposTotalVotesCount(),
-                                       [this](auto const &addr) { return dpos_eligible_vote_count_(addr); });
-
-  LOG(log_tr_) << "There are " << votes_.size() << " total votes in round " << round;
+  // Periodically verify unverified votes
+  vote_mgr_->verifyVotes(round, sortition_threshold_, getDposTotalVotesCount(),
```

Comment:
so, if I remember correctly one of the problems I pointed out is that inside `getVerifiedVotes` was called `getUnverifiedVotes()`, which would always make copies of votes. Because this is called very often it would reallocate Terabytes of memory evry hour. Now it is just moved to the `verifyVotes` if I see correctly. What happened with this PR, which was supposed to fix it ? 
https://github.com/Taraxa-project/taraxa-node/pull/927

Reply:
yes we did and we all agreed that we should not be making copies of votes here but we still do so what did you want to say by "Have discussed at the meeting" in terms of solving this ?

Reply:
Fixed by future_unverifiable_votes, verified_votes and getUnverifiedVotes return std::vector<std::shared_ptr<Vote>> in verifyVotes